datasets/bomen/acc/maatregelregistratie/v1.1.0.json

```diff
@@ -253,7 +253,7 @@
           "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
         ]
       },
-      "objectBeginTijd": {
+      "objectbegintijd": {
```

Comment:
Checkvraag: Ik zou hier ook camelcase verwachten, net als bij de aanpassingen van de eerdere velden. Of is dit hier bewust weggelaten?

Reply:
Hoi Jan, in de laatste versies van alle tabellen is het `objectbegintijd` , dus aan elkaar. voorheen stond het verkeerd.